lib/internal/errors.js

```diff
@@ -901,8 +901,13 @@ function determineSpecificType(value) {
  * @returns {string}
  */
 function formatList(array, type = 'and') {
-  return array.length < 3 ? ArrayPrototypeJoin(array, ` ${type} `) :
-    `${ArrayPrototypeJoin(ArrayPrototypeSlice(array, 0, -1), ', ')}, ${type} ${array[array.length - 1]}`;
+  switch (array.length) {
+    case 0: return '';
+    case 1: return array[0];
```

Comment:
```suggestion
    case 1: return `${array[0]}`;
```
Is this value guaranteed to always be a string? If not, we should probably keep the coercion.

Reply:
better safe than sorry